__tests__/app.test.js

```diff
@@ -59,20 +73,77 @@ describe("GET /api/articles/:article_id", () => {
             const article = response.body.article;
             //console.log(article);
             expect(typeof article).toBe("object");
+            expect(article).toHaveProperty("article_id");
+            expect(typeof article.article_id).toBe("number");
+            expect(article.article_id).toBe(1);
             expect(article).toHaveProperty("title");
             expect(typeof article.title).toBe("string");
+            expect(article.title).toBe("Living in the shadow of a great man");
             expect(article).toHaveProperty("topic");
             expect(typeof article.topic).toBe("string");
+            expect(article.topic).toBe("mitch");
             expect(article).toHaveProperty("author");
             expect(typeof article.author).toBe("string");
+            expect(article.author).toBe("butter_bridge");
             expect(article).toHaveProperty("body");
             expect(typeof article.body).toBe("string");
+            expect(article.body).toBe("I find this existence challenging");
             expect(article).toHaveProperty("created_at");
             expect(typeof article.created_at).toBe("string");
+            expect(article.created_at).toBe("2020-07-09T20:11:00.000Z");
             expect(article).toHaveProperty("votes");
             expect(typeof article.votes).toBe("number");
+            expect(article.votes).toBe(100);
             expect(article).toHaveProperty("article_img_url");
             expect(typeof article.article_img_url).toBe("string");
+            expect(article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
         })
-    })
+    });
+    test("Should respond with 400 and send an error response in case of an error", () => {
+        jest.spyOn(models, 'fetchArticlesById').mockRejectedValue();
+        return request(app).get("/api/articles/1").expect(400).then((response) => {
+            const text = response.error.text;
+            expect(text).toEqual("{\"status\":400,\"error\":\"Invalid query\"}");
+        });
+    });
```

Comment:
Nice 400 test, please add a 404 test for when an article with a valid but non-existent id is requested
Like `/api/articles/9999` for example